status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.125.4",
-    "commit-sha1": "7e1a894ab824904f0c90acbc307955940ef17800",
-    "src-sha256": "0sf33afgvmn4rj6ka7rxx3jnvpyjxcayv25jyiv1s9dkh581zy1i"
+    "version": "2c38b9e673dac2a795d9885b709ec424504120dd",
```

Comment:
There was a breaking change introduced in status-go, are you sure you want to update to latest status-go just yet?
https://github.com/status-im/status-go/pull/3133

Reply:
Oh, i noticed it before, let's solve it first

Reply:
I think its patched here : https://github.com/status-im/status-mobile/blob/01523f3a1de24b97425ff00e6fa89811956139e1/modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java#L1308 
but I'd like to ask @rasom to confirm, since he first reported it

Reply:
we can update the status-go version, the broken version is already in :) the PR fixing it is out https://github.com/status-im/status-go/pull/3145 , so should be making it to develop soon

Reply:
Thanks for confirming @cammellos !